template/node/Dockerfile

```diff
@@ -12,27 +12,31 @@ WORKDIR /root/
 # Turn down the verbosity to default level.
 ENV NPM_CONFIG_LOGLEVEL warn
 
+RUN mkdir -p /usr/local/app/function
```

Comment:
What about a regular home directory like `/home/app/` ?

Reply:
sounds good

Reply:
This should be: `/home/app/` as per the comment, could you update? I'd like to move forward and merge your change. Thank you for helping lead this